### DISCUSSION TITLE
Can I set a download location?

### DISCUSSION THREAD

**Discussion by nerdawitz** [pystardust/ani-cli#285] [Q&A] [answered]
so, can i? right now, it downloads onto the main drive in my home directory. I was wondering if i can set a download location like, download in a folder or onto a different mounted drive.

**Comment by ghost** [comment]
> so, can i? right now, it downloads onto the main drive in my home directory. I was wondering if i can set a download location like, download in a folder or onto a different mounted drive. 

I think you can, with the ``-d`` flag

**Reply by nerdawitz** [reply]
?? how? i know i can do this "ani-cli -d [anime name]" and then i choose the episodes i wanna download, and it downloads them to the main home directory. I'm asking if i can get it to download them to a different drive that's mounted by setting a download path

**Reply by dr-miami** [reply]
there is a way, you gotta cd in the download path and then you type the command, as an example:
```
cd Videos/Anime/
ani-cli -d -q 1080 -e 8 supermarket
```

**Reply by port19x** [reply]
> there is a way, you gotta cd in the download path and then you type the command, as an example:
> 
> ```
> cd Videos/Anime/
> ani-cli -d -q 1080 -e 8 supermarket
> ```

Wow, this is an old one.
We now have `ANI_CLI_DOWNLOAD_DIR`

**Comment by port19x** [comment, marked as answer]
`ani-cli -p download-dir anime-name`

**Reply by nerdawitz** [reply]
no, doesn't work. just says something about installing "aria2c"

**Reply by nerdawitz** [reply]
okay, so I got somewhere with it, I installed aria 2, and now I can proceed with the command you mentioned. 
but I get this error message, "Exception caught
Exception: [download_helper.cc:451] errorCode=1 Unrecognized URI or unsupported protocol:" and then it says download failed
